tests/test_regrid.py

```diff
@@ -567,12 +567,12 @@ def setup(self):
     def test_grid_missing_axis(self):
         ds = fixtures.generate_dataset(True, True)
 
-        ds_no_lat = ds.drop_vars(["lat"])
+        ds_no_lat = ds.drop_dims(["lat"])
 
         with pytest.raises(KeyError):
             ds_no_lat.regridder.grid
 
-        ds_no_lon = ds.drop_vars(["lon"])
```

Comment:
We have to drop the axis's dimension because its dimension can still exist without coordinates.

The axis should not exist if the corresponding dimension does not exist.